clearPort5005.sh

```diff
@@ -0,0 +1 @@
+sudo fuser -k 5005/tcp
```

Comment:
Should be moved into general-purpose "scripts" folder (along with potentially all other *.sh files currently in the top-level directory right now).